test/integration/workarea/kount_moneris_integration_test.rb

```diff
@@ -13,13 +13,7 @@ def test_new_credit_card
             ip_address: '170.115.187.68'
           })
 
-          response = Kount::OrderFraudService.new(
-            order: checkout.order,
-            payment: checkout.payment,
-            shippings: checkout.shippings
-          ).perform!
-
-          assert response.success?
+          refute(checkout.order.fraud_suspected_at?)
```

Comment:
To really test this properly, I need to do `assert(checkout.place_order)`, but I have to recreate the VCR cassette in order to do this and I don't have any Moneris credentials. So this is somewhat of a TODO...